add new document for v2 quickstart (#104)
opendatahub-io/distributed-workloads: Quick-Start-ODH-V2.md

Once your notebook environment is ready, in order to test our CodeFlare stack we will want to run though some of the demo notebooks provided by the CodeFlare community. So let's start by cloning their repo into our working environment.

```bash
git clone https://github.com/project-codeflare/codeflare-sdk
cd codeflare-sdk
```

We will rely on this demo code to train an mnist model. So feel free to open `codeflare-sdk/demo-notebooks/batch-job/batch_mnist.ipynb` to follow along instead.

### Run the Guided Demo Notebooks

There are a number of guided demos you can follow to become familiar with the CodeFlare-SDK and the CodeFlare stack.  Navigate to the path: `codeflare-sdk/demo-notebooks/guided-demos` to see and run the latest demos.

## Cleaning up the CodeFlare Install
To completely clean up all the CodeFlare components after an install, follow these steps:

1.  No appwrappers should be left running:
    ```bash
    oc get appwrappers -A
    ```
     If any are left, you'd want to delete them

2. Remove the notebook and notebook pvc:
   ```bash
   oc delete notebook jupyter-nb-kube-3aadmin -n opendatahub
   oc delete pvc jupyterhub-nb-kube-3aadmin-pvc -n opendatahub
   ```

> 3. Remove the codeflare-stack kfdef: (Removes MCAD, InstaScale, KubeRay and the Notebook image)

Reviewer: ```suggestion
3. Remove the codeflare-stack: (Removes MCAD, InstaScale, KubeRay and the Notebook image)
```

This stack integrates well with [Open Data Hub](https://opendatahub.io/), and helps to bring batch workloads, jobs, and queuing to the Data Science platform.

## Automatic deployment

As a quick alternative to the following manual deployment steps an automaic *makefile* script can be used to deploy the CodeFlare stack. This script also deploys the prerequisite operators and the entire CodeFlare stack up to the step [Submit your first job](#submit-your-first-job).
To use this script, clone the repo and execute:

```bash
make all-in-one
```

> Note : Execute ```make help``` to list additional available operations.

## Prerequisites

### Resources

> In addition to the resources required by the odh-core deployment, you will need the following to deploy the Distributed Workloads stack infrastructure pods:

Reviewer: `In addition to the resources required by the odh-core deployment` - this sentence refers to the KfDef, may be good to reword it to something like `In addition to the resources required by default ODH deployments`.

### CodeFlare Operator

The CodeFlare operator must be installed from the OperatorHub on your OpenShift cluster. The default settings will
suffice.

### NFD and GPU Operators

If you want to run GPU enabled workloads, you will need to install the [Node Feature Discovery Operator](https://github.com/openshift/cluster-nfd-operator) and the [NVIDIA GPU Operator](https://github.com/NVIDIA/gpu-operator) from the OperatorHub. For instructions on how to install and configure these operators, we recommend [this guide](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/steps-overview.html#high-level-steps).

## Creating K8s resources

1. Create the opendatahub namespace with the following command:

    ```bash
    oc create ns opendatahub
    ```

1. Create a datascience cluster with CodeFlare and Ray enabled:

    ```bash

>     oc apply -f https://raw.githubusercontent.com/opendatahub-io/distributed-workloads/main/codeflare-dsc.yaml -n opendatahub

Reviewer: ```suggestion
    oc apply -f https://raw.githubusercontent.com/opendatahub-io/distributed-workloads/main/codeflare-dsc.yaml
```
DSC CR is cluster wide, there is no need to provide namespace.

    ```

> Applying the above kfdef will result in the following objects being added to your cluster:

Reviewer: ```suggestion
Applying the above DataScienceCluster will result in the following objects being added to your cluster (apart from default ODH objects):
```

Author: I fixed the kfdef reference, but I wasn't sure about the statement you put in parenthesis. It felt awkward to me here. Do you think it is necessary?

Reviewer: no, it is just a suggestion, main point was the kfdef reference.
